ceph/ceph_admin/__init__.py

```diff
@@ -86,11 +87,8 @@ def distribute_cephadm_gen_pub_key(self, nodes=None):
     def set_tool_repo(self):
         """Add the given repo on every node part of the cluster."""
         for node in self.cluster.get_nodes():
-            node.exec_command(
-                sudo=True,
-                cmd="yum-config-manager --add-repo"
-                " {}compose/Tools/x86_64/os/".format(self.config.get("base_url")),
-            )
+            repo = path.join(self.config.get("base_url"), "compose/Tools/x86_64/os/")
```

Comment:
IMO, let's not use `path.join` instead check for the `/` ending. For example
```
base_url = self.config.get("base_url")
if not base_url.endswith('/'): base_url + '/'
cmd = f"yum-config-manager --add-repo {base_url}"
```

Reply:
But I'm thinking `os.path.join` looks like easy way, where it avoids if block and would be easy to merge multiple dir names